src/components/CodeRevokeModal/index.jsx

```diff
@@ -232,36 +228,34 @@ class CodeRevokeModal extends React.Component {
     } = this.state;
 
     return (
-      <>
-        <Modal
-          ref={this.modalRef}
-          dialogClassName="code-revoke"
-          title={this.renderTitle()}
-          body={this.renderBody()}
-          buttons={[
-            <Button
-              key="revoke-submit-btn"
-              disabled={submitting}
-              className="code-revoke-save-btn"
-              onClick={handleSubmit(this.handleModalSubmit)}
-            >
-              <>
-                {mode === MODAL_TYPES.revoke && submitting && <Icon className="fa fa-spinner fa-spin mr-2" />}
-                Revoke
-              </>
-            </Button>,
-            <SaveTemplateButton
-              key="save-revoke-template-btn"
-              templateType={MODAL_TYPES.revoke}
-              setMode={this.setMode}
-              handleSubmit={handleSubmit}
-              disabled={doNotEmail}
-            />,
-          ]}
-          onClose={onClose}
-          open
-        />
-      </>
+      <Modal
+        ref={this.modalRef}
+        dialogClassName="code-revoke"
+        title={this.renderTitle()}
+        body={this.renderBody()}
+        buttons={[
+          <Button
+            key="revoke-submit-btn"
+            disabled={submitting}
+            className="code-revoke-save-btn"
+            onClick={handleSubmit(this.handleModalSubmit)}
+          >
+            <>
```

Comment:
Same comment as above, maybe replace with a div or something?

Reply:
@kiram15 Same comment as above, back at ya ;) This use of a fragment is valid, whereas the top-level fragment's usage was not.

Reply:
@kiram15 And just like the other above comment's response, I take this one back too ;) I've since removed the fragment locally.